schemas/artifactpackaged.json

```
@@ -0,0 +1,75 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/cdevents/sdk-go/pkg/api/artifact-packaged-event",
+  "properties": {
+    "context": {
+      "properties": {
+        "version": {
+          "type": "string",
+          "enum": [
+            "draft"
+          ],
+          "default": "draft"
+        },
+        "id": {
+          "type": "string",
+          "minLength": 1
+        },
+        "source": {
+          "type": "string",
+          "minLength": 1
+        },
+        "type": {
+          "type": "string",
+          "minLength": 1
```

The same way as done on the version field, we could hard code the type to its explicit event type
```suggestion
          "enum": [
            "dev.cdevents.artifact.packaged"
          ],
          "default": "dev.cdevents.artifact.packaged"
```

---

Uhm, it sounds like a good idea, I will do that.

---

If that's ok, I will postpone that to the PR where I introduce the versioning.

---

Sure. I'll add a comment about that in the versioning issue.